### ISSUE TITLE
Regression in build model from URDF.

### ISSUE THREAD

**Issue by nim65s** [stack-of-tasks/pinocchio#1150]
Hi,

If I try to load [baxter_simple](https://github.com/humanoid-path-planner/hpp-environments/blob/master/urdf/tests/baxter_simple.urdf), it works on pinocchio 2.3.1 and fails on pinocchio 2.4.0 with:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ValueError: Model does not have any body named left_hand
```

To reproduce this, you can run:
```
apt update -qqy && apt install -qqy robotpkg-py36-hpp-environments
python3 -c 'import pinocchio; pinocchio.buildModelFromUrdf("/opt/openrobots/share/hpp_environments/urdf/tests/baxter_simple.urdf"); print("ok")'
```

in `docker run --rm -it gepetto/utils:pinocchio-2.3.1` / `2.4.0` (they are coming from this [Dockerfile](https://github.com/nim65s/gepetto-utils/blob/master/docker/pinocchio/Dockerfile) with `--build-arg BRANCH=v2.3.1` / `2.4.0`).

Did anything change with respect to the particular name `left_hand` for links ? It seems that everything else loads well on this URDF.

This issue follows https://github.com/humanoid-path-planner/hpp-pinocchio/issues/116

**Comment by jcarpent** [comment]
@nim65s I think that #1151 solves the issue. Sorry for the regression. After merging #1151, I will provide a new release.

**Comment by nim65s** [comment]
Wow, that was fast, thanks a lot !

**Comment by jcarpent** [comment]
Fixed by #1151.
